migrations/versions/4bca517e56a0_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
```

Comment:
Try to remember to add a message when you create a migration. It can make it easier to know which migration files to look in if there's a problem going on during setup.